### PR TITLE
Vpa adjustments

### PR DESCRIPTION
**Pull request by silenteh**
With this PR we are:
- setting a minimum amount of resources VPA is allow to scale down the daemonset.
- setting the limits to twice the requests so that the VPA can keep the ration when scaling

### CODE REVIEW

File: helm/aws-cloud-controller-manager-app/templates/vpa.yaml [hunk 1]
```diff
@@ -10,8 +10,9 @@ spec:
   resourcePolicy:
     containerPolicies:
     - containerName: {{ .Chart.Name }}
-      controlledValues: RequestsAndLimits
```
Comment (whites11) [inline helm/aws-cloud-controller-manager-app/templates/vpa.yaml:13]: why did you remove these?

Comment (silenteh) [reply helm/aws-cloud-controller-manager-app/templates/vpa.yaml:13]: Reverted them back

Comment (whites11) [reply helm/aws-cloud-controller-manager-app/templates/vpa.yaml:13]: thx

**Review by whites11 — APPROVED** [review]

**Comment by fiunchinho** [comment]
It's always good to have these changes in the changelog.
